app/api/help_seekers/entities.rb

```diff
@@ -6,6 +6,8 @@ class Volunteer < Common::Entities::User
     class Need < Common::Entities::Need
       expose :chosen_by, using: Volunteer, expose_nil: true
       expose :reviews, using: Common::Entities::Review
+      expose :status_updated_at
+      expose :updated_by
```

Comment:
We can expose them under the common namespace as they are useful for both volunteers and help seekers namespaces.